### PR TITLE
Redesign send confirmation

### PR DESCRIPTION
**Pull request by musidlo**

### CODE REVIEW

File: packages/app/features/send/confirm/screen.tsx [hunk 1]
```diff
@@ -225,143 +228,211 @@ export function SendConfirm() {
...
+          bg={'$color1'}
+          br={'$6'}
+          p={'$6'}
+          gap={'$4.5'}
+          $gtSm={{
+            gap: '$5',
+          }}
+        >
+          <XStack gap={'$4'} ai={'center'}>
+            <AvatarProfile profile={profile} size={'$3'} mx="none" circular />
```
Comment (youngkidwarrior) [inline packages/app/features/send/confirm/screen.tsx:253]: Lets use `<LinkableAvatar/>` here as well

File: packages/app/features/send/components/SendTopNav.tsx [hunk 2]
```diff
@@ -75,7 +75,7 @@ export function SendTopNav() {
             {(() => {
               switch (true) {
                 case path.includes('/confirm'):
-                  return 'Preview And Send'
+                  return 'Recipient'
```
Comment (youngkidwarrior) [inline packages/app/features/send/components/SendTopNav.tsx:78]: Designs are a little off here. Let's change this to Review Transfer and remove the other instance of it

File: packages/app/features/send/confirm/screen.tsx [hunk 3]
```diff
@@ -225,143 +228,211 @@ export function SendConfirm() {
...
-                {formatAmount(formatUnits(usdcFees.baseFee + usdcFees.gasFees, usdcFees.decimals))}{' '}
-                USDC
-              </Paragraph>
-            )}
-            {usdcFeesError && (
-              <Paragraph color="$error">{usdcFeesError?.message?.split('.').at(0)}</Paragraph>
-            )}
+      <YStack gap={'$4'}>
+        <Paragraph size={'$8'} mt={'$4'}>
+          Review transfer
```
Comment (youngkidwarrior) [inline packages/app/features/send/confirm/screen.tsx:241]: Remove this

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]

File: packages/app/features/send/confirm/screen.tsx (hunk 1 above)
Comment (musidlo) [reply packages/app/features/send/confirm/screen.tsx:253]: Applied

File: packages/app/features/send/components/SendTopNav.tsx (hunk 2 above)
Comment (musidlo) [reply packages/app/features/send/components/SendTopNav.tsx:78]: applied what was discussed on qa channel

File: packages/app/features/send/confirm/screen.tsx (hunk 3 above)
Comment (musidlo) [reply packages/app/features/send/confirm/screen.tsx:241]: Redesign send confirmation by musidlo · Pull Request #969 · 0xsend_sendapp

**Comment by youngkidwarrior** [comment]
* **#969** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/969?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/969?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `dev`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Review by youngkidwarrior — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Dec 29, 5:36 PM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/969).

**Pull request merged**